app/cluster/integration_test.go

```diff
@@ -212,7 +210,9 @@ func TestDynamicClusterManager(t *testing.T) {
 		rsources.NewNoOpService(),
 		destinationdebugger.NewNoOpService(),
 		transformationdebugger.NewNoOpService(),
-		processor.WithFeaturesRetryMaxAttempts(0))
+		[]enricher.PipelineEnricher{},
```

Comment:
```suggestion
		nil,
```

Reply:
I like passing empty array as it tells what argument it is in the calling argument. As per the code the treatment is same. What do you think ?